data/spells/scripts/monster/white_shade_paralyze.lua

```diff
@@ -0,0 +1,12 @@
+local condition = Condition(CONDITION_PARALYZE)
+condition:setParameter(CONDITION_PARAM_TICKS, 20000)
+condition:setFormula(-0.3, 0, -0.45, 0)
+
+local combat = Combat()
+combat:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_HITAREA)
+combat:setArea(createCombatArea(AREA_CIRCLE2X2))
+combat:addCondition(condition)
+
+function onCastSpell(creature, var)
+	return combat:execute(creature, var)
+end
```

Comment:
this spell is not needed in lua, you can use this in xml:

```
		<attack name="speed" interval="2000" chance="15" target="1" radius="2" speedchange="-450" duration="20000">
			<attribute key="areaEffect" value="blackspark" />
		</attack>
```
(don't forget to change spaces to tab when copying from github